docs/guides/1238-project-file-organization.md

```diff
@@ -0,0 +1,242 @@
+---
+slug: /1238/project-file-organization
+displayed_sidebar: "0.2"
+---
+
+# Project file organization
+
+As time will pass, your actions will get bigger and bigger. You will feel the need to better organize your file structure by moving some parts in subfolders.
+
+The easiest way, when iterating fast, is to rely on the root module.
+
+## CUE Module
+
+A module in CUE is any directory including a `cue.mod` folder. It makes for the prefix/root of importable packages (e.g, `universe.dagger.io` is a module)
+
+### Anonymous module (default)
+
+The default module name is an empty string, which means it's anonymous. That's when you don't need to import other packages inside your module. It represents most cases where someone has a Dagger plan and imports everything from third-party packages, like universe.
```

Comment:
`which means that it's an anonymous module`

`Anonymous modules are used when we don't need to import other packages. All Dagger configs start here: a plan represented by a single cue file, e.g. plan.cue, which imports everything from third-party packages, like universe.dagger.io/docker`.